sensor_msgs/Image image
geometry_msgs/Point[] shape
---
string                face_id
